### ISSUE TITLE
mono failed to build on debian sid

### ISSUE THREAD

**Issue by sherpya** [Linuxbrew/linuxbrew#871]
https://gist.github.com/anonymous/118b82cbffcdde3474b3

looks like mono installs fine, but fails on fsharp

```
==> make install
==> Cloning https://github.com/fsharp/fsharp.git
Updating /home/sherpya/.cache/Homebrew/mono--fsharp--git
==> Checking out tag 4.0.1.0
==> ./autogen.sh --prefix=/home/sherpya/.linuxbrew/Cellar/mono/4.2.2.30
==> make
Last 15 lines from /home/sherpya/.cache/Homebrew/Logs/mono/05.make:
         0 Warning(s)
         1 Error(s)

Time Elapsed 00:00:02.4085010
Makefile:9: recipe for target 'build-proto' failed
make[3]: *** [build-proto] Error 1
make[3]: Leaving directory '/tmp/mono--fsharp20160301-17865-17msfyb/src/fsharp/FSharp.Build-proto'
Makefile:15: recipe for target 'build-proto' failed
make[2]: *** [build-proto] Error 2
make[2]: Leaving directory '/tmp/mono--fsharp20160301-17865-17msfyb/src/fsharp'
Makefile:10: recipe for target 'all' failed
make[1]: *** [all] Error 2
make[1]: Leaving directory '/tmp/mono--fsharp20160301-17865-17msfyb/src/fsharp'
Makefile:4: recipe for target 'all' failed
make: *** [all] Error 2

READ THIS: https://github.com/Linuxbrew/linuxbrew/blob/master/share/doc/homebrew/Troubleshooting.md#troubleshooting
```

**Comment by sjackman** [comment]
```
Project "/tmp/mono--fsharp20160301-17865-17msfyb/src/fsharp/FSharp.Build-proto/FSharp.Build-proto.fsproj" (default target(s)):
	Target CallFsSrGen:
		Created directory "obj/proto/./"
		Tool /tmp/mono--fsharp20160301-17865-17msfyb/src/fsharp/FSharp.Build-proto/../../../lib/bootstrap/4.0/fssrgen.exe execution started with arguments:  /tmp/mono--fsharp20160301-17865-17msfyb/src/fsharp/FSharp.Build/FSBuild.txt  obj/proto/./FSBuild.fs  FSBuild.resx 
	Target BeforeBuild:
		Executing: mono --runtime=v4.0.30319 /tmp/mono--fsharp20160301-17865-17msfyb/src/fsharp/FSharp.Build-proto/../../../.nuget/NuGet.exe restore packages.config -PackagesDirectory packages -ConfigFile /tmp/mono--fsharp20160301-17865-17msfyb/src/fsharp/FSharp.Build-proto/../../../.nuget/NuGet.Config
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		WARNING: Error: SendFailure (Error writing headers)
		Unable to find version '2.0.3' of package 'FsCheck'.
		Unable to find version '2.6.4' of package 'NUnit.Runners'.
		Unable to find version '3.0.0' of package 'NUnit'.
		Unable to find version '3.0.0' of package 'NUnit.Console'.
/tmp/mono--fsharp20160301-17865-17msfyb/src/FSharpSource.targets: error : Command 'mono --runtime=v4.0.30319 /tmp/mono--fsharp20160301-17865-17msfyb/src/fsharp/FSharp.Build-proto/../../../.nuget/NuGet.exe restore packages.config -PackagesDirectory packages -ConfigFile /tmp/mono--fsharp20160301-17865-17msfyb/src/fsharp/FSharp.Build-proto/../../../.nuget/NuGet.Config' exited with code: 1.
	Task "Exec" execution -- FAILED
	Done building target "BeforeBuild" in project "/tmp/mono--fsharp20160301-17865-17msfyb/src/fsharp/FSharp.Build-proto/FSharp.Build-proto.fsproj".-- FAILED
Done building project "/tmp/mono--fsharp20160301-17865-17msfyb/src/fsharp/FSharp.Build-proto/FSharp.Build-proto.fsproj".-- FAILED
```
https://gist.githubusercontent.com/anonymous/118b82cbffcdde3474b3/raw/c33dd1ee55ef5920154a2dfa22452859a8d3ba45/05.make

**Comment by sjackman** [comment]
I'm afraid that I don't have the time available to troubleshoot this build failure myself. Hopefully someone else will stumble on this issue and be able to help you out.

**Comment by rwhogg** [comment]
I'm getting the same problem (Ubuntu 14.04).

I might have some time to look into this in the next few days.

**Comment by sjackman** [comment]
Great. Thanks, Bob. You've been a great help.

**Comment by rwhogg** [comment]
This might be related: https://github.com/fsharp/fsharp/issues/552

**Comment by rwhogg** [comment]
I also found https://github.com/Homebrew/homebrew/issues/49746, but as the error code reported is different, I'm hesitant to say they're related.

**Comment by rwhogg** [comment]
Think I may have found a fix. Looks like it's caused by missing SSL certificates making `nuget` unable to download packages correctly.

You have to do `mozroots --import --sync` before running `make` for fsharp. I'll see if I can get a successful build and then make a PR when I get it working.

**Comment by sjackman** [comment]
Fixed by #901.

**Comment by sherpya** [comment]
sorry for delay guys, it works for mee to
